SuperdenseCoding/Tasks.qs

```diff
@@ -3,11 +3,16 @@
 
 namespace Quantum.Kata.SuperdenseCoding {
     
-    open Microsoft.Quantum.Diagnostics;
     open Microsoft.Quantum.Intrinsic;
-    open Microsoft.Quantum.Canon;
-    
+    open Microsoft.Quantum.Measurement;
     
+    /// # Summary
+    /// Data type that represents the message that is transmitted 
+    /// as part of the superdense coding protocol. 
+    /// It includes two classical bits (Bit1 and Bit2) of type Bool.   
+    newtype Message = (Bit1 : Bool, Bit2 : Bool);    
```

Comment:
Minor thought, but do we want to use the Alice and Bob naming scheme here as well?

Reply:
For the message? That to me seems more confusing rather than helpful, since it sounds like it implies that one bit belongs to Alice, and one to Bob.